client/components/edit/create/StateControls.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { Dispatch } from "react";
+
+interface StateControlsProp {
```

Comment:
We're eventually going to have an "edit state" dialog box, so I wonder if StateControls should be in a more generic location than /create.

Reply:
Made change